ui/Screen/Wallet.svelte

```diff
@@ -23,40 +21,43 @@
   import Transactions from "./Wallet/Transactions.svelte";
   import LinkAddress from "./Wallet/LinkAddress.svelte";
 
-  import {
-    EmptyState,
-    HorizontalMenu,
-    SidebarLayout,
-  } from "ui/DesignSystem/Component";
+  import { EmptyState, SidebarLayout, TabBar } from "ui/DesignSystem/Component";
   import { Icon } from "ui/DesignSystem/Primitive";
 
-  const topbarMenuItems = [
-    {
-      icon: Icon.Transactions,
-      title: "Transactions",
-      href: path.walletTransactions(),
-    },
-  ];
+  export let activeTab: router.WalletTab = "transactions";
```

Comment:
There’s no need for a default value. `activeTab` is always part of the route object.

Reply:
d6db3de1ce76d9f68c26258d62f72a94a2cf0db9